testutil/datagen/btc_transaction.go

```diff
@@ -375,9 +375,10 @@ func GenRandomBabylonTxPair() ([]*wire.MsgTx, *btctxformatter.RawBtcCheckpoint)
 
 	// fake a raw checkpoint
 	rawBTCCkpt := GetRandomRawBtcCheckpoint()
+	tag := []byte{1, 2, 3, 4}
```

Comment:
```suggestion
	tag := datagen.GenRandomByteArray(4)
```
will resolve the `TODO` below.